tracer/build/_build/Build.GitHub.cs

```diff
@@ -45,8 +45,10 @@ partial class Build
     [Parameter("Is the ChangeLog expected to change?", List = false)]
     readonly bool ExpectChangelogUpdate = true;
 
+    [Parameter("Git repository name", Name = "GITHUB_REPOSITORY_NAME")]
+    readonly string GitHubRepositoryName = "dd-trace-dotnet";
+
     const string GitHubRepositoryOwner = "DataDog";
-    const string GitHubRepositoryName = "dd-trace-dotnet";
     const string AzureDevopsOrganisation = "https://dev.azure.com/datadoghq";
     const int AzureDevopsConsolidatePipelineId = 54;
     static readonly Guid AzureDevopsProjectId = Guid.Parse("a51c4863-3eb4-4c5d-878a-58b41a049e4e");
```

Comment:
I think these two values will also need to be customised for forks too

Reply:
Yes, you are right
Added in 185a33d11b66f283490412e023dc10701378ae36